File: nusaplanner_app/lib/classes/todolist.dart
class Todolist {
  /* int id;
  int userId; */
  int planningLanguageOne;
  int planningLanguageTwo;
  int planningLanguageThree;
  int planningDocumentOne;
  int planningDocumentTwo;
  int planningDocumentThree;
  int planningBankAccountOne;
  int planningVisaOne;
  int planningVisaTwo;
  int planningAnpOne;
  int planningAnpTwo;
  int planningDepartureOne;
  int planningDepartureTwo;

  Todolist(
      {/* required this.id,
      required this.userId, */
      required this.planningLanguageOne,
      required this.planningLanguageTwo,
      required this.planningLanguageThree,
      required this.planningDocumentOne,
      required this.planningDocumentTwo,
      required this.planningDocumentThree,
      required this.planningBankAccountOne,
      required this.planningVisaOne,
      required this.planningVisaTwo,
      required this.planningAnpOne,
      required this.planningAnpTwo,
      required this.planningDepartureOne,
      required this.planningDepartureTwo});

  factory Todolist.fromJson(Map<String, dynamic> json) {
    return Todolist(
        /* id: json['id'],
        userId: json['userId'], */
        planningLanguageOne: json['languageOne'],
        planningLanguageTwo: json['languageTwo'],
        planningLanguageThree: json['languageThree'],
        planningDocumentOne: json['documentOne'],
        planningDocumentTwo: json['documentTwo'],
        planningDocumentThree: json['documentThree'],
        planningBankAccountOne: json['bankOne'],
        planningVisaOne: json['visaOne'],
        planningVisaTwo: json['visaTwo'],
        planningAnpOne: json['anpOne'],
        planningAnpTwo: json['anpTwo'],
        planningDepartureOne: json['depOne'],
        planningDepartureTwo: json['depTwo']);
  }
}

File: fakeapione/app/Models/Todolist.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Notifications\Notifiable;

class Todolist extends Model
{
    use HasFactory, Notifiable;

    /**
     * The attributes that are mass assignable.
     *
     * @var string[]
     */
    protected $fillable = [
        'planning_language_one',
        'planning_language_two',
        'planning_language_three',
        'planning_document_one',
        'planning_document_two',
        'planning_document_three',
        'planning_bankAccount_one',
        'planning_visa_one',
        'planning_visa_two',
        'planning_anp_one',
        'planning_anp_two',
        'planning_departure_one',
        'planning_departure_two',
    ];

    public function user() {
        return $this->belongsTo(User::class);
    }
}


